Cargo.toml

```diff
@@ -19,21 +19,28 @@ rustls = { version = "0.21.0", default-features = false }
 tokio = "1.0"
 tokio-rustls = { version = "0.24.0", default-features = false }
 webpki-roots = { version = "0.23", optional = true }
+futures-util = { version = "0.3" }
 
 [dev-dependencies]
 futures-util = { version = "0.3.1", default-features = false }
 hyper = { version = "0.14", features = ["full"] }
 rustls = { version = "0.21.0", default-features = false, features = ["tls12"] }
 rustls-pemfile = "1.0.0"
-tokio = { version = "1.0", features = ["io-std", "macros", "net", "rt-multi-thread"] }
+tokio = { version = "1.0", features = [
```

Comment:
This is a formatting only change, please leave it out since it's irrelevant to the other changes.